src/components/pages/[...params].js

```diff
@@ -4,11 +4,45 @@ import { useRouter } from 'next/router';
 import { Box } from '@/ui/Box';
 import { useCookiesWithOptions } from '@/hooks/useCookiesWithOptions';
 import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
-import { memo, useMemo } from 'react';
+import { memo, useMemo, useState } from 'react';
+import { generatePath, matchPath } from 'react-router';
+import { getRedirects } from '../../redirects';
+import {
+  useHandleWindowMessage,
+  WindowMessageTypes,
+} from '@/hooks/useHandleWindowMessage';
+import PageNotFound from '@/pages/404';
+import { useIsClient } from '@/hooks/useIsClient';
+import { isFeatureFlagEnabledInCookies } from '@/hooks/useFeatureFlag';
+import { Cookies } from 'react-cookie';
 
 const prefixWhenNotEmpty = (value, prefix) =>
   value ? `${prefix}${value}` : value;
 
+const getRedirect = (originalPath, environment, cookies) => {
+  return getRedirects(environment)
+    .map(({ source, destination, permanent, featureFlag }) => {
+      // Don't follow redirects that are behind a feature flag
+      if (featureFlag) {
+        const enabled = isFeatureFlagEnabledInCookies(featureFlag, cookies);
+        if (!enabled) {
+          return false;
+        }
+      }
+
+      // Check if the redirect source matches the current path
+      const match = matchPath(originalPath, { path: source, exact: true });
+      if (match) {
```

Comment:
```suggestion
      const isMatch = matchPath(originalPath, { path: source, exact: true });
      if (isMatch) {
```

Reply:
This is not a boolean though.

Reply:
See for example https://github.com/cultuurnet/udb3-frontend/pull/230/files/d7cf50cf95f215a21fb32adbe1298476204794f6#diff-2940335828d78d99c2e896068a755ca522bc64c04b2e68a778b32a9ec0559669R37 where it's used as an object.